shell/src/main/java/alluxio/cli/GetConfKey.java

```diff
@@ -32,6 +32,8 @@
       .put("AWS_SECRETKEY", "aws.secretKey")
       .put("FS_GCS_ACCESSKEYID", "fs.gcs.accessKeyId")
       .put("FS_GCS_SECRETACCESSKEY", "fs.gcs.secretAccessKey")
+      .put("FS_OBS_ACCESSKEY", "fs.obs.accessKey")
+      .put("FS_OBS_SECRETKEY", "fs.obs.secretKey")
```

Comment:
add `OBS_ENDPOINT`?

Reply:
`FS_OBS_ENDPOINT` can be correctly transformed into `fs.obs.endpoint`, so it does not need to be put into this map.